apis/apps/v1/componentdefinition_types.go

```diff
@@ -99,6 +99,11 @@ type ComponentDefinitionSpec struct {
 	// +optional
 	Provider string `json:"provider,omitempty"`
 
+	// Specifies a BackupPolicyTemplate name if data protection functionality is supported.
+	// Which will automatically create the corresponding backupPolicy.
+	// +optional
+	BackupPolicyTemplateName string `json:"backupPolicyTemplateName,omitempty"`
```

Comment:
It's recommended to bind backup policy templates to the component definition, and do not let the component definition depend on a backup policy template.